test/views/HaveAnyTrusteesViewSpec.scala

```diff
@@ -45,4 +49,12 @@ class HaveAnyTrusteesViewSpec extends YesNoViewBehaviours {
     behave like pageWithReturnLink(createView(), frontendAppConfig.managePensionsSchemeOverviewUrl.url)
   }
 
+  "HaveAnyTrustees view in check mode where hub enabled" must {
```

Comment:
This whole spoke is already behind the toggle. So there is no value in adding tests when the toggle is disabled since this page won't appear in that case